.gitignore

```diff
@@ -10,6 +10,7 @@ version.txt
 
 ### Gradle ###
 .gradle
+gradle.properties
```

Comment:
Why is there a `gradle.properties` at the root of `Orion` to now ignore?

Reply:
`gradle.properties` is used by gradle to read personal project properties which we don't want to commit to git (in our case Oracle maven user and password). Hence, it is in ignore list.